chapter17/pxd/My/List/Util.pm

```diff
@@ -0,0 +1,32 @@
+package My::List::Util;
```

Comment:
I don't know if we might have to come back to this at some point - if so you may want to move it to the modules directory.